features/step_definitions/email_custom_steps.rb

```diff
@@ -1,40 +1,59 @@
-Given /^(?:a clear email queue|no emails have been sent|the email queue is clear)$/ do
+Given "the email queue is clear" do
   reset_mailer
 end
 
-Then /^"([^\"]*)" should be emailed$/ do |user|
+Given "a locale with translated emails" do
+  language = Language.find_or_create_by(short: "new", name: "New")
```

Comment:
This is a completely new language to prevent the existing translations from interfering with the tests. Otherwise a test might break when a Phrase export changes a translation.